types/Config.js

```diff
@@ -19,6 +19,7 @@ export type HasteConfig = {|
 |};
 
 export type ReporterConfig = [string, Object];
+export type TransformerConfig = [string, Object];
```

Comment:
Maybe make this `{[regex: string]: Path | [Path, Object]}` straight away and just use `?TransformerConfig` below? That would eliminate some of the inconsistencies in `key` vs `regex`, `Path` vs `string`, although we'd still have plenty of them left in the rest of the config 😄

Reply:
I tried doing this but ran into weird type errors 😞. Maybe this will be addressed when changes are made to go to ts?

Reply:
This was just a small nit, it's fine like this for now - Jest 25 is scheduled to get a big config overhaul, which will hopefully clean up all of those things anyway :) Thanks for the PR!